packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.tsx

```diff
@@ -162,14 +162,17 @@ export class HoverCard extends BaseComponent<IHoverCardProps, IHoverCardState> {
 
   @autobind
   private _executeCardOpen(ev: MouseEvent) {
+    this._async.clearTimeout(this._openTimerId);
```

Comment:
Should we move `this._async.clearTimeout(this._dismissTimerId)` in `_cardOpen` here for consistency with `_executeCardDismiss`?